src/utils.js

```diff
@@ -5,3 +5,45 @@ export const useIsomorphicLayoutEffect = typeof window !== 'undefined' ? useLayo
 // useForceUpdate hook
 const forcedReducer = state => state + 1;
 export const useForceUpdate = () => useReducer(forcedReducer, 0)[1];
+
+export const createDeepProxy = (obj, affected) => {
+  const proxyMap = {};
+  const handler = {
+    get: (target, key) => {
+      if (!affected.has(target)) {
+        affected.set(target, [key]);
+      } else {
+        const used = affected.get(target);
+        if (!used.includes(key)) used.push(key);
+      }
+      const val = target[key];
+      if (typeof val !== 'object') {
```

Comment:
there is a few objects (like buitins) which count not be proxied.

Reply:
I see. I should somehow detect it and work it around. (Or, maybe just warn developers.)